core/templates/dev/head/pages/subtopic-viewer-page/subtopic-viewer-page.controller.ts

```diff
@@ -63,11 +60,10 @@ angular.module('oppia').directive('subtopicViewerPage', [
           $rootScope.loadingMessage = 'Loading';
           SubtopicViewerBackendApiService.fetchSubtopicData(
             ctrl.topicName, ctrl.subtopicId).then(
-            function(subtopicDataDict) {
-              ctrl.pageContents =
-                SubtitledHtmlObjectFactory.createFromBackendDict(
-                  subtopicDataDict.page_contents.subtitled_html);
-              ctrl.subtopicTitle = subtopicDataDict.subtopic_title;
+            function(subtopicDataObject) {
+              ctrl.pageContents = subtopicDataObject.getPageContents().
```

Comment:
```suggestion
              ctrl.pageContents = (
                subtopicDataObject.getPageContents().getSubtitledHtml());
```

Maybe we can do it this way?

Reply:
Ok, Done!